RNTester/js/examples/ActionSheetIOS/ActionSheetIOSExample.js

```diff
@@ -25,6 +25,7 @@ const ScreenshotManager = NativeModules.ScreenshotManager;
 const BUTTONS = ['Option 0', 'Option 1', 'Option 2', 'Delete', 'Cancel'];
 const DESTRUCTIVE_INDEX = 3;
 const CANCEL_INDEX = 4;
+const DISABLED_BUTTON_INDICES = [1.0, 2.0];
```

Comment:
Is there a reason why you have used the float numbers (not ints) in the example?

Reply:
Nope, will refactor it to ints, thanks!